src/pages/Data.js

```diff
@@ -89,10 +94,52 @@ const Data = (props) => {
         </div>
     )
 
+    let pageLinks = (
+        <div className="flex flex-col justify-center items-center">
+            <div id="external-links" class="flex flex-row w-1/2 justify-between">
+                <div><a class="text-blue-500" target="_blank" rel="noopener noreferrer" href={`https://www.ncbi.nlm.nih.gov/sra/?term=${sraAccession}`}>SRA Link</a></div>
+                <div><a class="text-blue-500" target="_blank" rel="noopener noreferrer" href={`https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=${sraAccession}`}>Trace Link</a></div>
+                <div><a class="text-blue-500" href={`https://s3.amazonaws.com/lovelywater/bam/${sraAccession}.bam`} download>Download BAM File</a></div>
+            </div>
+        </div>
+    )
+
     return (
-        <div class="m-6">
-            {searchBox}
-            {sraAccession ? reportContent : <div></div> }
+        <div>
+            <div className="h-screen w-screen flex flex-col items-center justify-center">
+            <img src="/serratus.jpg" className="invisible sm:visible opacity-75 sm:fixed" style={{objectFit: 'cover', minWidth: '100vh', minHeight: '100vh'}} />
+                <div class="w-3/4 h-full flex flex-col justify-center items-center z-10 bg-blue-400 border border-gray-600 rounded-lg shadow-2xl bg-opacity-25 mt-10 mb-32">
+                    <div className="w-5/6 bg-gray-400 h-32 border rounded-lg border-gray-600 shadow-xl p-1 -mt-2 z-20 m-1">
+                        {searchBox}
+                        {sraAccession ? pageLinks : <div></div> }
+                        <div class="w-full text-center text-xl">
+                            {entrezStudyName ? <div className="text-xl italic">Study: {entrezStudyName}</div> : <div></div>}
+                        </div>
+                    </div>
+                <div className="flex h-64 flex-col justify-end items-center w-5/6 bg-gray-400 border rounded-lg border-gray-600 shadow-xl m-1">
+                    <div class="w-1/2 h-64 overflow-y-auto">
```

Comment:
can we change `w-1/2` to `w-600`? this is the width defined for the SVG created by D3.

Reply:
used `w-auto` in new update.